app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -318,6 +339,40 @@ protected void onPostExecute(Boolean success) {
         license.setText(""); // fill in from network...
     }
 
+    /**
+     * Uses Fresco to load an image from Url
+     * @param imageUrl
+     * @param context
+     */
+    private void loadImageFromUrl(String imageUrl,
```

Comment:
We already have this function defined here. 

https://github.com/commons-app/apps-android-commons/blob/f12837650a945c7075813a3e2811f0009321dc0b/app/src/main/java/fr/free/nrw/commons/widget/PicOfDayAppWidget.java

Can you please extract it to a new class called `FrescoImageLoader` and use the function at both the places. :)

Reply:
Done. :)